src/main/java/com/shootdoori/match/controller/LoginController.java

```diff
@@ -13,29 +18,83 @@
+        HttpServletRequest request,
+        HttpServletResponse response
+    ) {
+        AuthToken token = authService.login(loginRequest, request);
+        setRefreshTokenCookie(response, token.refreshToken());
 
-//        // Refresh Token은 HttpOnly 쿠키에 담아 응답
-//        Cookie cookie = new Cookie("refreshToken", token.getRefreshToken());
-//        cookie.setHttpOnly(true);
-//        cookie.setSecure(true); // HTTPS 환경에서만 사용
-//        cookie.setPath("/");
-//        cookie.setMaxAge(....);
-//        response.addCookie(cookie);
-
-        return ResponseEntity.ok(token);
+        return ResponseEntity.ok(new AuthTokenResponse(token.accessToken()));
     }
 
     @PostMapping("/register")
-    public ResponseEntity<AuthToken> register(@RequestBody ProfileCreateRequest profileCreateRequest) {
-        AuthToken token = authService.register(profileCreateRequest);
-        return ResponseEntity.ok(token);
+    public ResponseEntity<AuthTokenResponse> register(
+        @RequestBody ProfileCreateRequest profileCreateRequest,
+        HttpServletRequest request,
+        HttpServletResponse response
+    ) {
+        AuthToken token = authService.register(profileCreateRequest, request);
+        setRefreshTokenCookie(response, token.refreshToken());
+
+        return ResponseEntity.ok(new AuthTokenResponse(token.accessToken()));
+    }
+
+    @PostMapping("/refresh")
+    public ResponseEntity<AuthTokenResponse> refresh(
+        @CookieValue("refreshToken") String refreshToken,
+        HttpServletResponse response
+    ) {
+        AuthToken newTokens = tokenRefreshService.refreshAccessToken(refreshToken);
+        setRefreshTokenCookie(response, newTokens.refreshToken());
+
+        return ResponseEntity.ok(new AuthTokenResponse(newTokens.accessToken()));
+    }
+
+    @PostMapping("/logout")
+    public ResponseEntity<Void> logout(
+        @CookieValue("refreshToken") String refreshToken,
+        HttpServletResponse response
+    ) {
+        authService.logout(refreshToken);
+        expireRefreshTokenCookie(response);
+
+        return ResponseEntity.ok().build();
+    }
+
+    @PostMapping("/logout-all")
+    public ResponseEntity<Void> logoutAll(
+        @LoginUser User user,
+        HttpServletResponse response
+    ) {
+        authService.logoutAll(user.getId());
+        expireRefreshTokenCookie(response);
+
+        return ResponseEntity.ok().build();
+    }
+
+    private void expireRefreshTokenCookie(HttpServletResponse response) {
+        Cookie cookie = new Cookie("refreshToken", null);
+        cookie.setMaxAge(0);
+        cookie.setPath("/");
+        response.addCookie(cookie);
+    }
+
+    private void setRefreshTokenCookie(HttpServletResponse response, String refreshToken) {
+        Cookie cookie = new Cookie("refreshToken", refreshToken);
+        cookie.setHttpOnly(true);
+        cookie.setSecure(true);
+        cookie.setPath("/");
+        cookie.setMaxAge(7 * 24 * 60 * 60);
```

Comment:
숫자 값은 상수값 처리가 적절하지 않을까 생각합니다 :D
```suggestion
        private final int COOKIE_MAX_AGE = 7 * 24 * 60 * 60;
        cookie.setMaxAge(COOKIE_MAX_AGE);
```

Reply:
반영했습니다 :D